src/main/charts/confluence/values.yaml

```diff
@@ -188,6 +188,14 @@ synchrony:
   # Confluence service will use to communicate directly with Synchrony, so the URL must be resovable both from inside and
   # outside the Kubernetes cluster.
   ingressUrl:
+  # -- Specifies a list of additional Java libraries that should be added to the Confluence container.
```

Comment:
Not related to this PR, but I realised when looking at the values for our `synchrony` definition that we do not include a `resources` definition for it as we do our other product `StatefulSet's`. I think we probably should, there are specifics that would be nice to tune for the synchrony pod i.e. JVM memory requirements and resource `request` and `limit` sizings, see here: https://github.com/atlassian-labs/data-center-helm-charts/blob/master/src/main/charts/confluence/values.yaml#L106

Reply:
Thanks @errcode1202, created https://bulldog.internal.atlassian.com/browse/DCKUBE-322